config/opal_mca.m4

```diff
@@ -103,10 +105,10 @@ AC_DEFUN([OPAL_MCA],[
                 type=$item
             fi
             if test -z $comp ; then
-                AS_VAR_SET([AS_TR_SH([DISABLE_$type])], [1])
+                eval AS_TR_SH([DISABLE_${type}])=1
```

Comment:
The whole point of AS_VAR_SET is to not need this eval behavior.  This is not the root cause; I may have done something else wrong, but this is not the fix.

Reply:
@bwbarrett check now.